src/components/DataDisplay/Tutorial/Tutorial.tsx

```diff
@@ -0,0 +1,69 @@
+import { forwardRef, ReactElement, useState } from 'react';
+
+import { useTutorialSteps } from '../../../providers/TutorialStepsProvider';
+import TutorialStart from './TutorialStart';
+import TutorialSteps from './TutorialSteps';
+
+export enum Steps {
+  NOTSTARTED = 'not-started',
```

Comment:
Maybe instead of including this enum in the component we could rename the `name` property of `IStep` to be `key` and then we create the enums when we are about to use the `Tutorial` component? 

The `key` property could then have the type of string.